helm-service/main.go

```diff
@@ -168,7 +168,7 @@ func _main(args []string, env envConfig) int {
 		log.Fatal(fmt.Sprintf("failed to check whether helm-service has admin right: %v", err))
 	}
 	if !adminRights {
-		log.Fatal("helm-service has insufficient RBAC rights.")
+		log.Println("Warning: helm-service has non-admin RBAC rights. Will continue to operate")
```

Comment:
Let's change this message to "Warning: helm-service is running without admin RBAC rights. See https://github.com/keptn/keptn/issues/3511 for details."